README.md

```diff
@@ -517,11 +517,101 @@ var config = require('amodro-config')
 
 Arguments to `modify`:
 
-* **contents**: String. File conents that may contain a config call.
+* **contents**: String. File contents that may contain a config call.
 * **onConfig**: Function. Function called when the first config call is found. It will be passed an Object which is the current config, and the onConfig function should return an Object to use as the new config that will be serialized into the contents, replacing the old config.
 
 Returns a String the contents with the config changes applied.
 
+### amodro-trace/parse
+
+This module helps to extract dependencies from a JS file, which is an AMD module - which is wrapped in a `require` or a `define` statement.
+
+Methods accept either a string with the source content or an object  with JavaScript AST. If you traverse the AST multiple times, you can improve performance by parsing the source just once by a parser producing output according to the [ESTree Spec](https://github.com/estree/estree). For example, [Esprima](https://github.com/jquery/esprima):
+
+```javascript
+var esprima = require('esprima');
+
+var astRoot = esprima.parse(fileContents, {
+  range: includeIndexBasedRangeLocation,
+  loc: includeLineAndColumnBasedLocation
```

Comment:
`includeIndexBasedRangeLocation` and `includeLineAndColumnBasedLocation` are mentioned in a couple of the comment blocks, but do not show up anywhere except in comment blocks. If we need range and loc set, then we should just use `true` here, or if it does not matter to our code, then the examples can just be `esprima.parse(fileContents)`.

I also think it is

Reply:
I'm removing the options as you suggested. They are not important for the amodro-trace interface.

It was a clumsy way to remind the user by just variable names without comments, that Esprima has options and some of them can help, if they need to connect the AST node with the source code location in an editor, or if they need to generate the source code from AST again. Esprima users will use Esprima documentation anyway.